src/server.c

```diff
@@ -2172,6 +2172,7 @@ void initServerConfig(void) {
     server.configfile = NULL;
     server.executable = NULL;
     server.arch_bits = (sizeof(long) == 8) ? 64 : 32;
+    server.dbgAssertKeysize = 0; /* Disabled by default */
```

Comment:
what is `dbgAssertKeysize`?
We use the underscore naming convention, so why switch to camelCase here? As far as I know, camelCase is typically reserved for global variables.